### PR TITLE
Alert button

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1. added alert and button component
2. added documentation of alert and button component

image

### CODE REVIEW

File: components/alert-button/alert.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    margin-left:0;
+    border:1px solid grey;
+    padding:25px;
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
+      
+}
+
+.alert{
+    width:400px;
```
Comment (sri-rishi) [inline components/alert-button/alert.css]: can use rem instead of px

File: components/alert-button/alert.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+alert-container{
+    display:flex;
+    flex-flow:wrap column;
+    align-items:flex-start;
+    margin-left:0;
+    border:1px solid grey;
+    padding:25px;
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
```
Comment (sri-rishi) [inline components/alert-button/alert.css]: use variable naming of colors

File: components/alert-button/alert.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+      
+}
+
+.alert{
+    width:400px;
+    padding: 10px;
+    text-align:left;
+    flex-basis:50px;
+    margin-bottom:0.5rem;
```
Comment (sri-rishi) [inline components/alert-button/alert.css:28]: Just a suggestion use px for units less than 1rem

File: components/alert-button/alert.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
+alert-container{
+    display:flex;
+    flex-flow:wrap column;
+    align-items:flex-start;
+    margin-left:0;
+    border:1px solid grey;
+    padding:25px;
```
Comment (mehulsatardekar) [inline components/alert-button/alert.css]: use rem if it is more than 10px

File: components/alert-button/alert.css (hunk 2 above)
Comment (mehulsatardekar) [inline components/alert-button/alert.css]: use CSS variables

File: components/alert-button/alert.css [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
+      
+}
+
+.alert{
+    width:400px;
+    padding: 10px;
+    text-align:left;
+    flex-basis:50px;
```
Comment (mehulsatardekar) [inline components/alert-button/alert.css]: use rem here.

File: components/alert-button/button.html [hunk 6]
```diff
@@ -0,0 +1,181 @@
...
+            <!------------------------------->
+            <!-----------buttons------------->
+            <!------------------------------->
+            <div class="buttons-container mt2 pb2">
+
+                <!-- solid buttons
+        1) Primary 
+        2) secodary 
+        3) success  
+        4) warning
```
Comment (sri-rishi) [inline components/alert-button/button.html]: remove comments before pushing

**Review by mehulsatardekar — CHANGES_REQUESTED** [review]
I would suggest to use **rem** as much as possible if the sizing is more than 10px. and use CSS variables to avoid repetition.

File: components/alert-button/alert.css [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+    padding:25px;
+    width:fit-content;
+    background-color: rgb(248, 244, 244);
+      
+}
+
+.alert{
+    width:400px;
+    padding: 10px;
+    text-align:left;
```
Comment (amantulsyan35) [inline components/alert-button/alert.css:26]: can use flex start or flex end property here

File: components/alert-button/alert.css [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+.warning-alert{
+    color:#92400e;
+    background-color:#fef3c7;
+    border:0.5px solid #fcd34d;
+}
+
+.info-alert{
+    color:#155e75;
+    background-color:#cffafe;
+    border:0.5px solid #67e8f9;
```
Comment (gauravsinhaweb) [inline components/alert-button/alert.css:65]: use variable  colors

File: components/alert-button/alert.css (hunk 1 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: ok ,i will change it to rem

File: components/alert-button/alert.html [hunk 9]
```diff
@@ -0,0 +1,84 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/main.css">
+    <link rel="stylesheet" href="/doc.css">
+    <link rel="stylesheet" href="/components/alert-button/alert.css">
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
+    <script src="https://kit.fontawesome.com/9c98d67da5.js" crossorigin="anonymous"></script>
```
Comment (gauravsinhaweb) [inline components/alert-button/alert.html:12]: script is repeating here.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]
Good to merge after the above changes. Nice work!

File: components/alert-button/alert.css (hunk 1 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: made the changes ,please approve ,thanks for reviews

File: components/alert-button/alert.css (hunk 3 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css:28]: ok thanks for suggestion

**Review by mehulsatardekar — APPROVED** [review]

File: components/alert-button/alert.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: done with changes

File: components/alert-button/button.html (hunk 6 above)
Comment (JuvitaSaini09) [reply components/alert-button/button.html]: done ,thankyou for review

File: components/alert-button/alert.css (hunk 4 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: i have made the changes ,thankyou for review ,please approve PR

File: components/alert-button/alert.css (hunk 2 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: done

File: components/alert-button/alert.css (hunk 5 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css]: i have changed the units,thankyou for the review

File: components/alert-button/alert.css (hunk 8 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css:65]: changed ,thnks for reviewing plz approve PR

**Review by sri-rishi — APPROVED** [review]
Sorry for late reply, You can merge now

File: components/alert-button/alert.html (hunk 9 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.html:12]: i have corrected it,thankyou for the review ,Please approve PR

File: components/alert-button/alert.css (hunk 7 above)
Comment (JuvitaSaini09) [reply components/alert-button/alert.css:26]: i have the changes ,thankyou for the review .Please Approve PR

File: components/alert-button/alert.css [hunk 10]
```diff
@@ -0,0 +1,78 @@
+:root{
+    --blue1:#1e40af;
+    --blue3:#dbeafe;
+    --blue4:#93c5fd;
+    --white-light:#d4d4d4;
+    --grey-light:#a3a3a3;
+    --black-light:#404040;
+    --white-less1:rgb(248, 244, 244);
```
Comment (parvez-rashid) [inline components/alert-button/alert.css:8]: Use consistent units for colors: either all hex or all rgb. (Hex is easier in my opinion).

**Pull request merged**